src/foam/nanos/auth/AgentAuthService.js

```diff
@@ -42,6 +42,27 @@ foam.INTERFACE({
         }
       ]
     },
+    {
+      name: 'canActAs',
+      javaReturns: 'boolean',
+      swiftReturns: 'Bool',
+      swiftThrows: true,
```

Comment:
Add an array of exceptions that might be thrown in javaThrows

Reply:
@zahnno will talk to you about this, have a few questions.

Reply:
Sorry, super redundant comment, second only to this one.

Reply:
Done